PHP 7.2 - Group imports with trailing comma support

SpacePossum opened the pull request:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2907

First round of support;
- fix SingleImportPerStatementFixer not handling trailing comma correctly
- additional tests for SingleImportPerStatementFixer
- additional test for OrderedImportsFixerTest

This doesn't solve all issues for PHP 7.2 support, but it is a good first round I think :)

julienfalque commented on tests/Fixer/Import/SingleImportPerStatementFixerTest.php:
```diff
@@ -272,6 +272,22 @@ public function provide70Cases()
         ];
     }
 
+    public function atestMessyComments()
```
Extra `a` at the beginning of the method name.

SpacePossum replied:
ah lol, yeah that is my poor-man way of disabling utest when debugging ;)  good catch

keradus replied:
hint: I do the same, but with a TODO prefix. it's easier to spot accidentally left one

SpacePossum commented:
good point @julienfalque , target branch changed to 2.2

SpacePossum commented:
what is up with appveyor now?

keradus commented:
Please add `tests/Fixtures/Integration/misc/PHP7_2.test` file containing part of 7.2 that are covered by this PR.

julienfalque commented:
This PR should be ready now.

SpacePossum commented:
thanks @julienfalque 
I've added more test code to the PHP 7.2 integration test, rebased to remove the merge before that commit  and squashed to one commit for each of us.

keradus commented:
Thank you @SpacePossum.
